lxd/container.go

```diff
@@ -166,10 +166,12 @@ func containerLXDCreateAsEmpty(d *Daemon, name string,
 		return nil, err
 	}
 
-	// Now create the empty storage
-	if err := c.Storage.ContainerCreate(c); err != nil {
-		c.Delete()
-		return nil, err
+	if !d.IsMock {
+		// Now create the empty storage
+		if err := c.Storage.ContainerCreate(c); err != nil {
+			c.Delete()
+			return nil, err
+		}
```

Comment:
Why can't the mock storage driver just not do anything on ContainerCreate(), avoiding the if statement here?